code/src/App.js

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import data from './data.json';
+import { Album } from "./components/Album";
+import { Name } from "./components/Name"; 
+import { Artist } from "./components/Artist";
 
 console.log(data);
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+
+      
+       {/* map out data.json so we can use "album" in Album.js*/ }
+      <section className="main-container">
+      <div className='header'> <h1> New Release </h1> </div> 
+      
+      <div className='album-container'>
+        {data.albums.items.map((album) => {
+          return <Album key={album.id} album={album} />;
+        })}
+      </div>
+      </section>  
+    </>
```

Comment:
Before I start, How amazing that  you submitted the project like half way through the week! I like the approach of the code here. This code part is easy to read. Nice ,simple display to be deployed to the html.